packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -545,6 +548,34 @@ function commitRoot(root: FiberRoot, finishedWork: Fiber): void {
       : updateExpirationTimeBeforeCommit;
   markCommittedPriorityLevels(root, earliestRemainingTimeBeforeCommit);
 
+  let prevInteractions: Set<Interaction> = (null: any);
+  let committedInteractions: Array<Interaction> = enableInteractionTracking
```

Comment:
Why are these outside the feature flag?

Reply:
So they can be referenced below when we restore interactions and call `onInteractionScheduledWorkCompleted`. If the feature flag is disabled, they should get removed anyway since they aren't referenced.